src/contracts/Protocol/BountyQuestion.sol

```diff
@@ -29,7 +29,7 @@ contract BountyQuestion is ERC721, ERC721Enumerable, ERC721URIStorage, ERC721Bur
     constructor() ERC721("MetricsDAO Question", "MDQ") {}
 
     // working standard metadata format:  Title, Description, Program
-    function safeMint(address to, string memory uri) public onlyOwner returns (uint256) {
+    function safeMint(address to, string calldata uri) public returns (uint256) {
```

Comment:
I think we want this to be `onlyAPI` because if someone calls `safeMint` but doesn't call initialize on the other controllers state will be out of whack